lib/turn.rb

```diff
@@ -1,2 +1,40 @@
 class Turn
+  attr_reader :player,
+              :round
+
+  def initialize(player, round)
+    @player = player
+    @round = [0..10]
+  end
+
+  def player_won_round
+    if @player.place_pins.results == [0, 1]
+      p "Look at you go!?"
+      p "-".times 10
+    end
+    if @player.place_pins.results == [0, 2]
+      p "You heating up but you won't do it again."
+      p "-".times 10
+    end
+    if @player.place_pins.results == [0, 3]
+      p "I let you have that one!"
+      p "-".times 10
+    end
+    if @player.place_pins.results == [0, 4]
```

Comment:
you would actually want [4,4] to be the final win condition since you need 4 pins in position and pins that are in position are also correct.

Reply:
if index 0 is "only correct" and index 1 is "both are correct" then shouldn't [0, 4] suffice?

Reply:
If [0, 4] then by default it's should be the same as [4, 4].

Reply:
Disregard my last two comments. I get what you're saying. I'll change it.